### PR TITLE
Gives admins a command to mass-delete objects in a given zone

### PR DESCRIPTION
**Pull request by ShiftyRail**
[administration] [qol]

Useful to clean-up objects lagging the server after a bus.
Tested.

### CODE REVIEW

File: code/modules/admin/admin_mass_del_in_zone.dm [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+	set desc="Delete all atoms of a given type in a zone."
+	set name="Mass delete in zone"
+	if (!check_rights(R_SERVER))
+		to_chat(src, "<span class='warning'>You need +SERVER to do this.</span>")
+		return FALSE
+
+	src = usr.client.holder // why lummox why
+	
+	var/list/dat = list()
+	dat += "<h3>Mass deletion in a zone</h3>"
```
Comment (ShiftyRail) [inline code/modules/admin/admin_mass_del_in_zone.dm]: This can actually be made into a single var, hang on

**Comment by Kurfursten** [comment]
Just for the record you can already do this (easier?) with buildmode, just turn on selective

**Comment by ShiftyRail** [comment]
True, but I've got buildmode-phobia, and it can also be used for larger areas (50x50 square without zooming out).
If you think these benefits do not outweigh the redundancy, feel free to close this, I don't mind

**Comment by Kurfursten** [comment]
I don't think more options is bad per se, although we definitely have some admin command bloat on our tabs. Ehhh. If the admin team wants it, far be it from me to deny it.

Someone has *got* to clean up our admin tabs though. They are just out of control. Some kind of UI.

File: code/modules/admin/admin_mass_del_in_zone.dm (hunk 1 above)
Comment (Exxion) [reply code/modules/admin/admin_mass_del_in_zone.dm]: Just use document syntax

**Pull request merged**